### DISCUSSION TITLE
Syntax can be made better

### DISCUSSION THREAD

**Discussion by sultanarun** [datapane/datapane#371] [Ideas]
### Goals

Please change the syntax to match that of streamlit. The current one is ugly.

### Non-Goals

1.
2.
3.

### Background

Please make it more easy

### Proposal

Look at the syntax of streamlit. Please make it simple like that.

**Comment by sultanarun** [comment]
I digged more into datapane and it is awesome. It is very lightweight compared to streamlit. The syntax is confusing but you will soon fall in love with it.

**Comment by mands** [comment]
Hi @sultanarun - glad you like the library! Would you be able to give some examples of syntax that you found confusing or areas that you'd like to see improvement? Thanks!
